sklearn/multiclass.py

```diff
@@ -1025,6 +1025,31 @@ def fit(self, X, y):
 
         return self
 
+    def predict_proba(self, X):
+        """Return faux probability estimates.
+
+        Estimates are based upon euclidean distance between the underlying
+        estimators' predictions and the class code.
+
+        Parameters
+        ----------
+        X : (sparse) array-like of shape (n_samples, n_features)
+            Data.
+
+        Returns
+        -------
+        T : (sparse) array-like of shape (n_samples, n_classes)
+            Returns the score of the sample for each class in the model,
+            where classes are ordered as they are in `self.classes_`.
+        """
+        check_is_fitted(self)
+        Y = np.array([_predict_binary(e, X) for e in self.estimators_]).T
+        dist = euclidean_distances(Y, self.code_book_)
+        # Inverse distance weighting:
+        eps = 1e-16
```

Comment:
This would start to be tricky if we deal with 32-bit floating point. I assume this is better to define with

```suggestion
        eps = np.finfo(dist.dtype).eps
```

Reply:
Nice, will do.